app.js

```diff
@@ -12,10 +12,12 @@ function copy({
   return fs.copy(sourcePath, destinationPath, { overwrite });
 }
 
-function createDirectory({
+async function createDirectory({
   path: directoryPath,
 }) {
-  return fs.mkdirs(directoryPath);
+  const createdPath = await fs.mkdirs(directoryPath);
+
+  return createdPath === undefined ? "Directory Already Exists" : "Directory Created";
```

Comment:
Are you sure it should be handled like that? What have you based this design on? Any docs?

According to their repo docs, it should accepts a callback.
https://github.com/jprichardson/node-fs-extra/blob/9.1.0/docs/ensureDir.md